_posts/2015-05-20-plugin-settings.markdown

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: Plugin Settings
+status: public
+type: post
+published: true
+author: Go Team
+---
+
+Go is continously improving its plugin infrastructure. Last year the Plugin API change from JAVA interface based approach to JSON message based one to support multiple versions of same end-point for backward compatibility is working out quiet well.
+
+Starting 15.2.0 Go will support "Plugin Settings" that will allow plugins to accept global settings from Admins. Currently these configurations had to be supported via system properties or a file that is in specified format in a specified location, which makes it a little hapazard. With this feature "all" plugins will have one approach to accept plugins settings from user & access plugin settings from Go Server.
```

Comment:
I think the explanation  "that will allow plugins to accept global settings from Admins" is vague. Can we highlight the fact about plugins being able to manage and store settings and about our support for that?

You could state that earlier the plugins had to manage configuration on their own. What we'll providing is an infrastructure for storing settings. Which will be persistent and can be backed up etc. 

"which makes it a little hapazard" (spell check)